pytolemaic/utils/general.py

```diff
@@ -37,7 +38,7 @@ class GeneralUtils():
 
     @classmethod
     def is_classification(cls, model):
-        return hasattr(model, 'predict_proba')
+        return is_classifier(model)
```

Comment:
Thanks for the feedback and PR!
However, I do not wish to add "_estimator_type" property to the API nor require the inheritance of a Mixin class. 
So I'll use is_classifier only if the model inherits a Mixin class as a way to accommodate regression models with predict_proba (which doesn't make much sense, is it? :) )

Reply:
@d4rk-lucif3r The new version is up, please check.